src/utils.ts

```diff
@@ -80,9 +80,22 @@ function isObjectEmpty (object: Object): boolean {
   return Object.keys(object).length === 0;
 }
 
+function replaceAll (input: string, search: string, replace: string) {
+  return input.split(search).join(replace);
+}
+
+function arnMatches (inputArn: string, actualArn: string) {
```

Comment:
Hmm not sure if `arnMatches` is a good name for this. It's being used to match names instead of ARNs. Also, its pretty generic so it could be used to match any string.

How about `wildcardMatches` instead or `isWildcardMatch`?

Reply:
Changed to `wildcardMatches` in 54a780d6f72b40a392cf5ae7a78193fc9285c919